### PR TITLE
Update debugger-for-edge.md

### PR DESCRIPTION
**Pull request by vchapel**
Changed title to reflect responsible team

**Rendered article for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio-code/debugger-for-edge?branch=pr-en-us-1559
**Live article:**
https://docs.microsoft.com/en-us/microsoft-edge/visual-studio-code/debugger-for-edge

### CODE REVIEW

File: microsoft-edge/visual-studio-code/debugger-for-edge.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 07/26/2021
+ms.date: 11/05/2021
 ms.topic: article
 ms.prod: microsoft-edge
 keywords: microsoft edge, web development, f12 tools, devtools, vs code, visual studio code, debugger
 ---
-# Debug Microsoft Edge from Visual Studio Code
+# Get in touch with the Debugger for Microsoft Edge Visual Studio Code Extension team
```
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/debugger-for-edge.md:11]: The article isn't about getting in touch.  The article is a high-level overview of this extension, with a link.

**Review by mikehoffms — CHANGES_REQUESTED** [review]
Not sure the intent of this PR.  Expected the body to be clarified about which team the reader would be getting in touch with, if that were discussed, but I don't see that discussed.

File: microsoft-edge/visual-studio-code/debugger-for-edge.md (hunk 1 above)
Comment (vchapel) [reply microsoft-edge/visual-studio-code/debugger-for-edge.md]: oudated

File: microsoft-edge/visual-studio-code/debugger-for-edge.md [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
 ms.prod: microsoft-edge
 keywords: microsoft edge, web development, f12 tools, devtools, vs code, visual studio code, debugger
 ---
-# Debug Microsoft Edge from Visual Studio Code
+# Debug Microsoft Edge in Visual Studio Code
 
-[Visual Studio Code](https://code.visualstudio.com) includes a built-in debugger for Microsoft Edge which can launch the browser or attach to an already running browser.
+[Visual Studio Code](https://code.visualstudio.com) includes a built-in debugger for Microsoft Edge, which can launch the browser or attach to an already running browser.
 
 This built-in debugger lets you debug your front-end JavaScript code line-by-line and see `console.log()` statements directly from [Visual Studio Code](https://code.visualstudio.com).
```
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/debugger-for-edge.md:15]:
```suggestion
This built-in debugger lets you debug your front-end JavaScript code line-by-line and see `console.log()` statements directly from Visual Studio Code.
```

File: microsoft-edge/visual-studio-code/debugger-for-edge.md [hunk 3]
```diff
@@ -1,17 +1,25 @@
...
-For more information, navigate to [Browser debugging in VS Code](https://code.visualstudio.com/docs/nodejs/browser-debugging).
+For more information, see [Browser debugging in VS Code](https://code.visualstudio.com/docs/nodejs/browser-debugging) and [Browser debugging with Microsoft Edge DevTools integration in Visual Studio Code](/microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension#browser-debugging-with-microsoft-edge-devtools-integration-in-visual-studio-code).
+
+
+<!-- ====================================================================== -->
+## Get in touch with the Debugger for Microsoft Edge Visual Studio Code Extension team
+
+Send your feedback by [filing an issue](https://github.com/Microsoft/vscode-edge-devtools/issues/new) in the `vscode-edge-devtools` repo.
+
+Your contributions are welcome, to help make the Microsoft Edge DevTools extension better.  Find everything you need to get started in the [vscode-edge-devtools](https://github.com/Microsoft/vscode-edge-devtools) repo.
```
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/debugger-for-edge.md:25]:
```suggestion
Your contributions are welcome, to help make the Microsoft Edge DevTools extension better.  Find everything you need to get started in the `vscode-edge-devtools` repo.
```

File: microsoft-edge/visual-studio-code/debugger-for-edge.md [hunk 4]
```diff
@@ -1,17 +1,25 @@
...
 This built-in debugger lets you debug your front-end JavaScript code line-by-line and see `console.log()` statements directly from [Visual Studio Code](https://code.visualstudio.com).
 
-For more information, navigate to [Browser debugging in VS Code](https://code.visualstudio.com/docs/nodejs/browser-debugging).
+For more information, see [Browser debugging in VS Code](https://code.visualstudio.com/docs/nodejs/browser-debugging) and [Browser debugging with Microsoft Edge DevTools integration in Visual Studio Code](/microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension#browser-debugging-with-microsoft-edge-devtools-integration-in-visual-studio-code).
+
+
+<!-- ====================================================================== -->
+## Get in touch with the Debugger for Microsoft Edge Visual Studio Code Extension team
+
+Send your feedback by [filing an issue](https://github.com/Microsoft/vscode-edge-devtools/issues/new) in the `vscode-edge-devtools` repo.
```
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/debugger-for-edge.md:23]:
```suggestion
Send your feedback by [filing an issue](https://github.com/Microsoft/vscode-edge-devtools/issues/new) in the [vscode-edge-devtools repo](https://github.com/Microsoft/vscode-edge-devtools).
```

**Review by mikehoffms — CHANGES_REQUESTED** [review]

File: microsoft-edge/visual-studio-code/debugger-for-edge.md (hunk 2 above)
Comment (vchapel) [reply microsoft-edge/visual-studio-code/debugger-for-edge.md:15]: +1

File: microsoft-edge/visual-studio-code/debugger-for-edge.md (hunk 3 above)
Comment (vchapel) [reply microsoft-edge/visual-studio-code/debugger-for-edge.md:25]: excellent

File: microsoft-edge/visual-studio-code/debugger-for-edge.md (hunk 4 above)
Comment (vchapel) [reply microsoft-edge/visual-studio-code/debugger-for-edge.md]: yes and yes

File: microsoft-edge/visual-studio-code/debugger-for-edge.md [hunk 5]
```diff
@@ -1,17 +1,25 @@
...
 
-[Visual Studio Code](https://code.visualstudio.com) includes a built-in debugger for Microsoft Edge which can launch the browser or attach to an already running browser.
+[Visual Studio Code](https://code.visualstudio.com) includes a built-in debugger for Microsoft Edge, which can launch the browser or attach to an already running browser.
 
-This built-in debugger lets you debug your front-end JavaScript code line-by-line and see `console.log()` statements directly from [Visual Studio Code](https://code.visualstudio.com).
+This built-in debugger lets you debug your front-end JavaScript code line-by-line and see `console.log()` statements directly from Visual Studio Code.
 
-For more information, navigate to [Browser debugging in VS Code](https://code.visualstudio.com/docs/nodejs/browser-debugging).
+For more information, see [Browser debugging in VS Code](https://code.visualstudio.com/docs/nodejs/browser-debugging) and [Browser debugging with Microsoft Edge DevTools integration in Visual Studio Code](/microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension#browser-debugging-with-microsoft-edge-devtools-integration-in-visual-studio-code).
```
Comment (captainbrosset) [inline microsoft-edge/visual-studio-code/debugger-for-edge.md:17]: The second link here is really about the DevTools extension in VS Code. It does talk a tiny bit about debugging JS from VS Code, but then goes on to talk about how you can open DevTools in VS Code by installing the DevTools extension.
So I'm not sure it actually helps users learn more information here.

I still think we need a link to it here. But maybe rephrase to something like:

```
For more information, see [Browser debugging in VS Code](link 1).

You can also inspect HTML and CSS using DevTools from within VS Code. To learn more, see [Browser debugging with Microsoft Edge Devtools integration in VS Code](link 2).
```

File: microsoft-edge/visual-studio-code/debugger-for-edge.md [hunk 6]
```diff
@@ -1,17 +1,25 @@
...
+
+
+<!-- ====================================================================== -->
+## Get in touch with the Debugger for Microsoft Edge Visual Studio Code Extension team
+
+Send your feedback by [filing an issue](https://github.com/Microsoft/vscode-edge-devtools/issues/new) in the [vscode-edge-devtools repo](https://github.com/Microsoft/vscode-edge-devtools).
```
Comment (captainbrosset) [inline microsoft-edge/visual-studio-code/debugger-for-edge.md:23]: This links to the devtools VS Code extension repo. The page about this extension is https://docs.microsoft.com/en-us/microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension
So that page should link to this repo (if it doesn't already do it).

The present page, however, is about the built-in JS debugger in VS Code. It is maintained by a different team, and its repo is this one: https://github.com/microsoft/vscode-js-debug

I think @codepo8 should decide what's the best thing to do here, but I would suggest:

```
## Get in touch with the js-debug team

Send your feedback by [filing an issue](https://github.com/microsoft/vscode-js-debug/issues/new) in the vscode-js-debug repository.
```

**Review by captainbrosset — CHANGES_REQUESTED** [review]

File: microsoft-edge/visual-studio-code/debugger-for-edge.md [hunk 7]
```diff
@@ -1,17 +1,25 @@
...
 
-This built-in debugger lets you debug your front-end JavaScript code line-by-line and see `console.log()` statements directly from [Visual Studio Code](https://code.visualstudio.com).
+This built-in debugger lets you debug your front-end JavaScript code line-by-line and see `console.log()` statements directly from Visual Studio Code.
 
-For more information, navigate to [Browser debugging in VS Code](https://code.visualstudio.com/docs/nodejs/browser-debugging).
+For more information, see [Browser debugging in VS Code](https://code.visualstudio.com/docs/nodejs/browser-debugging) and [Browser debugging with Microsoft Edge DevTools integration in Visual Studio Code](/microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension#browser-debugging-with-microsoft-edge-devtools-integration-in-visual-studio-code).
+
+
+<!-- ====================================================================== -->
+## Get in touch with the Debugger for Microsoft Edge Visual Studio Code Extension team
```
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/debugger-for-edge.md:21]: Can we please get rid of this giant, distracting h2 (looming over this short page) and just have a low-key link instead

**Review by mikehoffms — CHANGES_REQUESTED** [review]

File: microsoft-edge/visual-studio-code/debugger-for-edge.md (hunk 6 above)
Comment (mikehoffms) [reply microsoft-edge/visual-studio-code/debugger-for-edge.md:23]: Given that it's a short page, with no other h2 headings, I think there should be sentences with a link, but no h2 heading.

File: microsoft-edge/visual-studio-code/debugger-for-edge.md (hunk 5 above)
Comment (vchapel) [reply microsoft-edge/visual-studio-code/debugger-for-edge.md:17]: changed

File: microsoft-edge/visual-studio-code/debugger-for-edge.md (hunk 6 above)
Comment (vchapel) [reply microsoft-edge/visual-studio-code/debugger-for-edge.md:23]: changed

File: microsoft-edge/visual-studio-code/debugger-for-edge.md (hunk 7 above)
Comment (vchapel) [reply microsoft-edge/visual-studio-code/debugger-for-edge.md:21]: i tried just removing the h2 altogether ... see what you think.

File: microsoft-edge/visual-studio-code/debugger-for-edge.md [hunk 8]
```diff
@@ -1,17 +1,25 @@
 ---
-description: How to debug Microsoft Edge from Visual Studio Code
-title: Debug Microsoft Edge from Visual Studio Code
+description: You can debug Microsoft Edge from Visual Studio Code using the Debugger Extension
```
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/debugger-for-edge.md:2]: @captainbrosset - was your plan to remove the focus from the Debugger Extension and focus on the built-in capabilities of VS Code instead?

Comment (captainbrosset) [reply microsoft-edge/visual-studio-code/debugger-for-edge.md:2]: There is no such thing as the debugger extension anymore. So I think this description needs to be updated.
VS Code can debug JS code in Edge/Chrome/WebView/Node by default now. No need for an extension anymore, this is built-in.

File: microsoft-edge/visual-studio-code/debugger-for-edge.md [hunk 9]
```diff
@@ -1,17 +1,25 @@
...
-For more information, navigate to [Browser debugging in VS Code](https://code.visualstudio.com/docs/nodejs/browser-debugging).
+
+
+For more information, see [Browser debugging in VS Code](https://code.visualstudio.com/docs/nodejs/browser-debugging).
+
+You can also inspect HTML and CSS using DevTools from within VS Code. To learn more, see [Browser debugging with Microsoft Edge Devtools integration in VS Code](/microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension#browser-debugging-with-microsoft-edge-devtools-integration-in-visual-studio-code).
+
+Send your feedback by [filing an issue](https://github.com/microsoft/vscode-js-debug/issues/new) in the [vscode-js-debug repository](https://github.com/microsoft/vscode-js-debug).
+
+Your contributions are welcome, to help make the Microsoft Edge DevTools extension better.  Find everything you need to get started in the `vscode-edge-devtools` repo.
```
Comment (captainbrosset) [inline microsoft-edge/visual-studio-code/debugger-for-edge.md:25]: The page looks good to me now. I would only note that this last sentence about contributions being welcomed might be better suited here: https://review.docs.microsoft.com/en-us/microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension?branch=pr-en-us-1559#getting-in-touch-with-the-microsoft-edge-devtools-extension-team
As it turns out, it is already there. So I think we can probably drop it from here.

**Review by captainbrosset — APPROVED** [review]

File: microsoft-edge/visual-studio-code/debugger-for-edge.md [hunk 10]
```diff
@@ -1,17 +1,23 @@
...
+This built-in debugger lets you debug your front-end JavaScript code line-by-line and see `console.log()` statements directly from Visual Studio Code.
 
-For more information, navigate to [Browser debugging in VS Code](https://code.visualstudio.com/docs/nodejs/browser-debugging).
+
+
+For more information, see [Browser debugging in VS Code](https://code.visualstudio.com/docs/nodejs/browser-debugging).
+
+You can also inspect HTML and CSS using DevTools from within VS Code. To learn more, see [Browser debugging with Microsoft Edge Devtools integration in VS Code](/microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension#browser-debugging-with-microsoft-edge-devtools-integration-in-visual-studio-code).
+
+Send your feedback by [filing an issue](https://github.com/microsoft/vscode-js-debug/issues/new) in the [vscode-js-debug repository](https://github.com/microsoft/vscode-js-debug).
```
Comment (mikehoffms) [inline microsoft-edge/visual-studio-code/debugger-for-edge.md:23]: Possibly move this paragraph above the "You can also" paragraph.  The structure of the article right now is:
1. Tool A.
2. Tool B.
3. How to give feedback (about tool A).
And/or, to disambiguate which tool the feedback is about, change "Send your feedback..." to "Send your feedback about (tool A)..."

**Review by mikehoffms — APPROVED** [review]

File: microsoft-edge/visual-studio-code/debugger-for-edge.md (hunk 8 above)
Comment (vchapel) [reply microsoft-edge/visual-studio-code/debugger-for-edge.md]: done

File: microsoft-edge/visual-studio-code/debugger-for-edge.md (hunk 9 above)
Comment (vchapel) [reply microsoft-edge/visual-studio-code/debugger-for-edge.md]: done

**Review by codepo8 — APPROVED** [review]
LGTM
